src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,576 @@
+    </attribute>
+    <attribute code="0x001A" side="server" define="NIGHT_VISION" type="TriStateAutoEnum" min="0x00" max="0x02" writable="true" optional="true">
+      <description>NightVision</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001B" side="server" define="NIGHT_VISION_ILLUM" type="TriStateAutoEnum" min="0x00" max="0x02" writable="true" optional="true">
+      <description>NightVisionIllum</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001C" side="server" define="AWB" type="boolean" writable="true" optional="true">
+      <description>AWB</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001D" side="server" define="SHUTTER_SPEED" type="boolean" writable="true" optional="true">
+      <description>ShutterSpeed</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001E" side="server" define="ISO" type="boolean" writable="true" optional="true">
+      <description>ISO</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001F" side="server" define="VIEWPORT" type="ViewportStruct" optional="true">
+      <description>Viewport</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0020" side="server" define="SPEAKER_ENABLED" type="boolean" writable="true" optional="true">
+      <description>SpeakerEnabled</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0021" side="server" define="SPEAKER_VOLUME_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpeakerVolumeLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0022" side="server" define="SPEAKER_MAX_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpeakerMaxLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0023" side="server" define="SPEAKER_MIN_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpeakerMinLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0024" side="server" define="MIC_STATUS" type="boolean" writable="true" optional="true">
```

Comment:
Spec issue: What does a boolean "MicStatus" mean?  What does true mean?  What does false mean?

This needs a better name.

Reply:
Captured comment in spec.